tfs/reader.py

```diff
@@ -21,17 +21,28 @@
 
 
 def read_tfs(
-    tfs_file_path: Union[pathlib.Path, str], index: str = None, non_unique_behavior: str = "warn"
+    tfs_file_path: Union[pathlib.Path, str],
+    index: str = None,
+    non_unique_behavior: str = "warn",
+    validate_after_reading: bool = True,
```

Comment:
why not just `validate`? it's in the reader anyway.

Reply:
Because then we have a name collision with the `validate` function inside this scope. I'm open to other names, but I don't think it's too big a deal (and auto-completion exists).